Readme.md

```diff
@@ -61,8 +61,10 @@ When an error occurs, the response is returned in JSON format with the following
 - taskID (required): Integer value representing the unique task identifier.
 - overwrite (optional): Boolean value indicating whether to overwrite an existing task directory.
 - description (required): File upload for the task description (e.g., description.pdf). 
-- inputFiles (required): Array of file uploads representing the input files.
-- outputFiles (required): Array of file uploads representing the output files.
+- archive (required): Archive file (.zip or .tar.gz) with the following folder structure after decompressing:
+    - Task - directory that should contain the description.pdf file
```

Comment:
we do we have 1 folder inside the archive? Isn't it just better to have archive in which you have description and two folders with inputs and outputs? Because later in method 4. Get Task Files. you return the archive without unnessesary nesting. I think we don't need this additional layer